scripts/pin_the_slow_way.py

```diff
@@ -41,7 +41,7 @@
           'glib': 'glib 2.51.*',  # 2.50.2
           'gmp': 'gmp >=5.0.1,<7', # 6.1.0
           'harfbuzz': 'harfbuzz 1.3.*',  # 0.9.39
-          'hdf5': 'hdf5 1.8.17|1.8.17.*',  # 1.8.17
+          'hdf5': 'hdf5 1.8.18|1.8.18.*',  # 1.8.17
```

Comment:
@qwhelan are you referring to `defaults` or `conda-forge`? I am confused b/c you linked the PR to `conda-forge`, but that was merged and we **are** moving to the latest `hdf5` thanks to you.